website/content/docs/secrets/pki/cmpv2.mdx

```diff
@@ -0,0 +1,130 @@
+---
+layout: docs
+page_title: Certificate Management Protocol v2 (CMPv2) within Vault | PKI - Secrets Engines
+description: An overview of the Enrollment over Secure Transport protocol implementation within Vault.
+---
+
+# PKI secrets engine - Certificate Management Protocol v2 (CMPv2) <EnterpriseAlert inline="true" />
+
+This document covers configuration and limitations of Vault's PKI Secrets Engine
+implementation of the [CMPv2 protocol](https://datatracker.ietf.org/doc/html/rfc4210) <EnterpriseAlert inline="true" />.
+
+## What is Certificate Management Protocol v2 (CMPv2)?
+
+The CMP protocol is an IETF standardized protocol, [RFC 4210](https://datatracker.ietf.org/doc/html/rfc4210),
+that allows clients to acquire client certificates and their associated Certificate
+Authority (CA) certficates.
+
+## Enabling CMPv2 support on a Vault PKI  mount
+
+To configure an existing mount to serve EST clients, the following steps are
+required, which are broken down into three main categories:
+
+ 1. [Authentication mechanisms](#configuring-est-authentication)
+ 1. [Updating PKI tunable parameters](#updating-the-pki-mount-tunable-parameters)
+ 1. [PKI CMPv2 configuration](#pki-cmpv2-configuration)
+
+### Configuring CMPv2 Authentication
+
+At this time, Vault's implementation of CMPv2 supports only 
+[Certificate TLS authentication](/vault/docs/auth/cert), where clients proof
+of posession of a TLS client certificate authenticates them to Vault.
+
+Authentication leverages a separate Vault authentication
+mount, within the same namespace, to validate the client provided credentials
+along with the client's ACL policy to enforce. 
+
+For proper accounting, a mount supporting CMPv2 authentication should be
+dedicated to this purpose, not shared with other workflows.  In other words,
+create a new certificate auth mount for CMPv2 even if you already have one 
+another in use for other purposes.
+
+When setting up the authentication mount for CMPv2 clients, the token type must
+be configured to return [batch tokens](/vault/docs/concepts/tokens#batch-tokens).
+Batch tokens are required to avoid an excessive amount of leases being generated
+and persisted as every CMPv2 incoming request needs to be authenticated.
+
+The path within an ACL policy must match the `cmp` path underneath the 
+PKI mount.  The path to use can be the default `cmp` path or a role based one.
+
+If using the `sign-verbatim` as a path policy, the following
+ACL policy will allow an authenticated client access the required PKI CMP path.
```

Comment:
```suggestion
For example, if you use the `sign-verbatim` as a path policy, you can define
an ACL policy using the default PKI CMP path, `pki/cmp`:
```

Style correction: write in active voice, speak directly to the reader with "you" for instructions.